transport_plugins/bled112/iotile_transport_bled112/virtual_bled112.py

```diff
@@ -108,6 +115,46 @@ def __init__(self, args):
             self.stop_sync()
             raise
 
+    def init_virtual_device_info(self, args):
+        self.virtual_info['advertising_version'] = int(args.get('advertising_version', 1),0)
+        if self.virtual_info['advertising_version'] not in (1, 2):
+            raise ArgumentError("Invalid advertising version specified in args",
+            supported=(1, 2), found=self.virtual_info['advertising_version'])
+
+        self.virtual_info['reboot_count'] = int(args.get('reboot_count', 1), 0)
+        if self.virtual_info['reboot_count'] <= 0:
+            raise ArgumentError("Reboot count must be greater than 0.",
+            supported="> 0", found=self.virtual_info['reboot_count'])
+
+        self.virtual_info['broadcast_stream'] = int(args.get('broadcast_stream', 0xDEAD), 0)
```

Comment:
`broadcast_stream` and `broadcast_value` should come from the virtual device that is being served rather than the virtual interface.  By default nothing is broadcast but if the virtual_device broadcasts a value it is saved in the virtual interface and used to update the advertising packet.  

See: https://github.com/iotile/coretools/blob/master/transport_plugins/bled112/iotile_transport_bled112/virtual_bled112.py#L187

for how the `broadcast_reading` is naturally updated when a device that is being served streams a broadcast reading.